File: Java/035_SearchInsertPosition.java
/*************************************************************************
    > File Name: 035_SearchInsertPosition.java
    > Author: daiyu
	> Url: https://leetcode.com/problems/search-insert-position/
    > Created Time: 二  9/22 09:50:52 2015
 ************************************************************************/
public class Solution {
    public int searchInsert(int[] nums, int target) {
        int result = 0;
        
        for (int i = 0; i < nums.length; i ++) {
            if (nums[i] >= target) {
                return i;
            } else {
                result += 1;
                continue;
            }
        }
        
        
        return result;
    }
}

File: Python/035_SearchInsertPosition.py
class Solution(object):
    def searchInsert(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: int
        """
        index = 0
        for i in range(len(nums)):
            if nums[i] < target:
                continue
            else:
                index = i
                break
        # 如果所有的值都小于target
        if nums[len(nums) - 1] < target:
            index = len(nums)
            
        return index
                